Heroku/backend/database/build.gradle.kts

```diff
@@ -28,6 +28,8 @@ dependencies {
 		// Except for ComparisonFailure, which is provided by :test-helpers.
 		exclude(group = libs.test.junit.vintage.get().module.group, module = libs.test.junit.vintage.get().module.name)
 	}
+	// TODEL https://github.com/shazam/shazamcrest/pull/39
+	testImplementation(enforcedPlatform("com.google.code.gson:gson:2.8.9"))
```

Comment:
`net.twisterrob.cinema.database.services.ViewServiceIntgTest#addView returns each piece of the View(Session)` failed
```
java.lang.NoSuchMethodError: 'void com.google.gson.internal.ConstructorConstructor.<init>(java.util.Map)'
	at com.google.gson.graph.GraphAdapterBuilder.<init>(GraphAdapterBuilder.java:36)
	at com.shazam.shazamcrest.matcher.GsonProvider.registerCircularReferenceTypes(GsonProvider.java:136)
	at com.shazam.shazamcrest.matcher.GsonProvider.gson(GsonProvider.java:60)
	at com.shazam.shazamcrest.matcher.DiagnosingCustomisableMatcher.matches(DiagnosingCustomisableMatcher.java:66)
	at org.hamcrest.DiagnosingMatcher.matches(DiagnosingMatcher.java:12)
	at com.shazam.shazamcrest.MatcherAssert.assertThat(MatcherAssert.java:40)
	at com.shazam.shazamcrest.MatcherAssert.assertThat(MatcherAssert.java:29)
	at net.twisterrob.cinema.database.services.ViewServiceIntgTest.addView returns each piece of the View(ViewServiceIntgTest.kt:66)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
```